packages/hadron-app-registry/src/app-registry.ts

```diff
@@ -122,7 +148,7 @@ class AppRegistry {
    *
    * @returns {Array} The container components.
    */
-  getContainer(name) {
+  getContainer(name: string): unknown[] {
```

Comment:
From a quick search in the code base this method might be unused? Maybe we can just drop it.